modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js

```diff
@@ -262,7 +265,22 @@ const formatRules = (pages, rules) => {
 	return formattedRules;
 };
 
-const fieldNameBelongsToAction = (fieldName, actions) => {
+const expressionHasNonNumericFields = (action, pages) => {
+	const expressionFields = getExpressionFields(action);
+	let hasNonNumericFields = false;
+
+	if (expressionFields && expressionFields.length > 0) {
+		expressionFields.forEach((field) => {
+			if (getFieldType(field, pages) !== 'numeric') {
```

Comment:
@alinedoleron, we agreed to invoke the new `getPages` method here instead of invoke on method `fieldNameBelongsToAction`, right?